### PR TITLE
Docs: Update menu.yaml

### PR DESCRIPTION
**Pull request by oddlittlebird**
Fixes a typo

### CODE REVIEW

**Review by marefr — APPROVED** [review]
LGTM

File: docs/sources/guides/whats-new-in-v7-0.md [hunk 1]
```diff
@@ -185,7 +185,7 @@ It was released as a beta feature in Grafana 6.7. The feedback has been really p
 
 ## Time zone support
 
-You can now override the [time zone]({{< relref "../reference/timerange/#time-zone-options" >}}) used to display date and time values in a dashboard. One benefit of this is that you can specify the local time zone of the service or system that you are monitoring which can be helpful when monitoring a system or service that operates across several time zones.
+You can now override the [time zone]({{< relref "../dashboards/time-range-controls.md" >}}) used to display date and time values in a dashboard. One benefit of this is that you can specify the local time zone of the service or system that you are monitoring which can be helpful when monitoring a system or service that operates across several time zones.
```
Comment (marefr) [inline docs/sources/guides/whats-new-in-v7-0.md:188]: Already a PR open to resolve this: https://github.com/grafana/grafana/pull/24794

**Review by marefr — DISMISSED** [review]
See comment

File: docs/sources/guides/whats-new-in-v7-0.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/guides/whats-new-in-v7-0.md:188]: Yeah, but I can't get this one to pass the build tests unless I fix the link.

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**